osquery/core/windows/ntapi.h

```diff
@@ -86,13 +122,26 @@ typedef NTSTATUS(WINAPI* NTQUERYSYMBOLICLINKOBJECT)(HANDLE LinkHandle,
                                                     PUNICODE_STRING LinkTarget,
                                                     PULONG ReturnedLength);
 
-typedef struct _SYSTEM_HANDLE_INFORMATION {
-  ULONG ProcessId;
-  BYTE ObjectTypeNumber;
-  BYTE Flags;
-  USHORT Handle;
-  PVOID Object;
-  ACCESS_MASK GrantedAccess;
+typedef struct __PUBLIC_OBJECT_TYPE_INFORMATION {
+    UNICODE_STRING TypeName;
+    ULONG Reserved [22];    // reserved for internal use
+} PUBLIC_OBJECT_TYPE_INFORMATION, *PPUBLIC_OBJECT_TYPE_INFORMATION;
+
+typedef struct _SYSTEM_HANDLE_TABLE_ENTRY_INFO
+{
+    USHORT UniqueProcessId;
```

Comment:
[FYI] This is limited to max USHORT (65,535). I've come across problems using this api (https://github.com/boostorg/process/issues/200) when the PID exceeds the value of 65,535. If you can avoid using `SYSTEM_HANDLE_INFORMATION` and instead use `SYSTEM_HANDLE_INFORMATION_EX` if possible.

Reply:
Done. I'm  now using `SYSTEM_HANDLE_INFORMATION_EX`. however I kept the definiton of `SYSTEM_HANDLE_INFORMATION` in `osquery/core/windows/ntapi.h` if someone else need it